include/Song.h

```diff
@@ -97,6 +97,7 @@ class EXPORT Song : public TrackContainer
 
 
 	void processNextBuffer();
+	void processAutomations(const TrackList& tracks, MidiTime timeStart, fpp_t frames, int tcoNum);
```

Comment:
Why cannot `processAutomations()` be private until there is a real need to be public? May I do the cut+paste?